python/lgbserver/lgbserver/test_model.py

```diff
@@ -46,20 +47,30 @@ def test_model():
                'petal_width_(cm)': {0: 0.2}, 'sepal_length_(cm)': {0: 5.1}}
 
     response = model.predict({"inputs": [request, request]})
-    assert numpy.argmax(response["predictions"][0]) == 2
+    assert numpy.argmax(response["predictions"][0]) == 0
```

Comment:
@sivanantha321 why the result is changed?

Reply:
ah I see it was changed in previous PR  
https://github.com/kserve/kserve/commit/7baebda3ea4b2d856550e77eb0ee6e36da5d6fc5